examples/gaiad/node.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+)
+
+const (
+	flagWithTendermint = "with-tendermint"
+)
+
+var (
+	initNodeCmd = &cobra.Command{
+		Use:   "init <flags???>",
```

Comment:
remove `<flags??>` this is what it prints:
```
Initialize full node

Usage:
  gaiad init <flags???> [flags]

Flags:
  -h, --help   help for init
```

Reply:
Yup. Placeholder for me to add real flags. Will do

Reply:
Oh are we going to say which flags are mandatory in here?? That would be cool. I'm also in favour of just including the fact that they're mandatory in the flag description